ui/deps.edn

```diff
@@ -85,15 +85,17 @@
   :local-deps
   {:override-deps
    {
-    is.d0x/cljs-web3-next {:local/root "../../d0x-libs/shared/cljs-web3-next"}
-    district0x/district-ui-graphql {:local/root "../../district-ui-graphql"}
-    is.d0x/district-ui-bundle {:local/root "../../d0x-libs/browser/district-ui-bundle"}
-    is.d0x/district-ui-smart-contracts {:local/root "../../d0x-libs/browser/district-ui-smart-contracts"}
-    is.d0x/district-ui-router {:local/root "../../d0x-libs/browser/district-ui-router"}
-    is.d0x/district-ui-web3-accounts {:local/root "../../d0x-libs/browser/district-ui-web3-accounts"}
-    is.d0x/re-frame-web3-fx {:local/root "../../d0x-libs/browser/re-frame-web3-fx"}
-    is.d0x/cljs-ipfs-http-client {:local/root "../../d0x-libs/shared/cljs-ipfs-http-client"}
-    is.d0x/district-ui-web3-tx {:local/root "../../d0x-libs/browser/district-ui-web3-tx"}}}
+    ;; is.d0x/cljs-web3-next {:local/root "../../d0x-libs/shared/cljs-web3-next"}
+    ;; district0x/district-ui-graphql {:local/root "../../district-ui-graphql"}
+    ;; is.d0x/district-ui-bundle {:local/root "../../d0x-libs/browser/district-ui-bundle"}
+    ;; is.d0x/district-ui-smart-contracts {:local/root "../../d0x-libs/browser/district-ui-smart-contracts"}
+    ;; is.d0x/district-ui-router {:local/root "../../d0x-libs/browser/district-ui-router"}
+    ;; is.d0x/district-ui-web3-accounts {:local/root "../../d0x-libs/browser/district-ui-web3-accounts"}
+    ;; is.d0x/re-frame-web3-fx {:local/root "../../d0x-libs/browser/re-frame-web3-fx"}
+    ;; is.d0x/cljs-ipfs-http-client {:local/root "../../d0x-libs/shared/cljs-ipfs-http-client"}
+    ;; is.d0x/district-ui-web3-tx {:local/root "../../d0x-libs/browser/district-ui-web3-tx"}
```

Comment:
These shouldn't be commented out. They're under the alias `:local-deps` and will be used only when compilation is started with this alias, e.g. like I do in the development 
  - https://github.com/district0x/ethlance/blob/newlance/Procfile#L5-L6
  - https://github.com/district0x/ethlance/blob/newlance/bb.edn#L37-L43